WordPress/Classes/ViewRelated/Site Creation/Final Assembly/LandInTheEditorHelper.swift

```diff
@@ -6,12 +6,7 @@ class LandInTheEditorHelper {
     /// - Parameter navigationController: UINavigationController used to present the home page editor
     /// - Parameter completion: HomepageEditorCompletion callback to be invoked after the user finishes editing the home page, or immediately iwhen the feature flag is disabled
     static func landInTheEditorOrContinue(for blog: Blog, navigationController: UINavigationController, completion: @escaping HomepageEditorCompletion) {
-        // branch here for feature flag
-        if FeatureFlag.landInTheEditor.enabled {
```

Comment:
The flag was off, but it seems the branch that was kept was the "flag-enabled" branch. 
I suggest removing the function the wraps this too since it does nothing if the flag is off.